seatunnel-transforms/seatunnel-transforms-spark/seatunnel-transform-spark-etl/src/main/scala/org/apache/seatunnel/spark/transform/service/EtlHandler.scala

```diff
@@ -0,0 +1,8 @@
+package org.apache.seatunnel.spark.transform.service
+
```

Comment:
@Interest1-wyt Please add the license header, thanks

Reply:
ok i'll add it later :)